docs/samples/box/shadow.md

```diff
@@ -0,0 +1,169 @@
+# Shadow
+
+```js chart-editor
+// <block:setup:4>
+const DATA_COUNT = 8;
+const MIN = [0, 50];
+const MAX = [50, 100];
+
+Utils.srand(8);
+
+const data = {
+  datasets: [{
+    data: Utils.points({count: DATA_COUNT, min: MIN[0], max: MAX[0]})
+  }, {
+    data: Utils.points({count: DATA_COUNT, min: MIN[1], max: MAX[1]})
+  }]
+};
+// </block:setup>
+
+// <block:annotation1:1>
+const annotation1 = {
+  type: 'box',
+  backgroundColor: 'rgba(0,150,0,0.02)',
```

Comment:
Spaces after comma

Reply:
@LeeLenaleee do you mean inside of color definitions?

Reply:
yes so instead of `backgroundColor: 'rgba(0,150,0,0.02)'` you get `backgroundColor: 'rgba(0, 150, 0, 0.02)',`. 

Note the space in front of the `150`, `0` and `0.02`

Reply:
OK! as for `var`, this should be applied to all samples in order to keep them aligned. I'm going to submit a PR for that as well

Reply:
done!